packages/components/src/components/FiltersForm.vue

```diff
@@ -56,9 +63,25 @@ export default {
     return {
       showSuggestions: false,
       inputValue: '',
+      selectedSuggestion: null,
     };
   },
 
+  watch: {
+    selectedSuggestion: {
+      handler(selectedSuggestion) {
+        if (selectedSuggestion === null) {
+          return;
+        }
+
+        this.$refs.suggestionsList.scrollTop =
+          this.$refs.suggestionsList.querySelectorAll('li')[
+            selectedSuggestion
+          ].offsetTop;
```

Comment:
Can this element fall in the center of the dropdown whenever possible? It's better UX to see both what's above and below instead of only seeing items below the current selection.

![image](https://user-images.githubusercontent.com/8737782/146237055-17d9241c-f0c6-4a42-9019-f8534fc9b88b.png)

Reply:
Done https://github.com/applandinc/appmap-js/pull/492/commits/cbf18694cd0b8b40b732ee6cb70a73eb4d426f30